integration/data/test_restore_inc.py

```diff
@@ -116,9 +116,40 @@ def restore_inc_test(grpc_engine_manager,  # NOQA
     verify_no_frontend_data(grpc_engine_manager,
                             0, snap0_data, grpc_dr_controller)
 
-    # mock restore crash/error
     delta_file1 = "volume-delta-" + backup0_name + ".img"
     if "vfs" in backup_target:
+        # mock restore crash/error: cannot clean up the delta file path
```

Comment:
Mock similar case for restore volume tests

Reply:
There is no test for the regular restore volume.

Reply:
There is no test? https://github.com/longhorn/longhorn-engine/blob/master/integration/data/test_backup.py#L61

Reply:
Sorry for missing this... 
But we can test the regular restore command in this function, too.

Reply:
Sorry for missing this... 
But we can test the regular restore command in this function, too.